pkg/gen/filters/common/arrays.go

```diff
@@ -9,6 +9,11 @@ func Contains(a []any, s string) bool {
 	return false
 }
 
+func AppendStringList(list []string, s string) []string {
```

Comment:
This should be just called AppendList. At the end the function just returns the result of append. Why not expose append directly as common filter?